docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,59 @@
+## Checklist
+<!--
+Mark an `[x]` for completed items.
+-->
+
+* [ ] Update CHANGELOG.rst  for each commit separately.
+* [ ] Bump the process version:
+    * **Last number**: changes that do not affect the api/interface. Examples:
+        bugfix, typo fix, change/add warning messages,..
+    * **Middle number**: bigger changes that don't make it backwards incompatible.
+        Examples: add output field, add non-mandatory input parameter, use
+        a different tool that produces same results...
+    * **First number**: Backwards incompatible changes, are changes that
+        break the api/interface. Examples: renaming the input/output, adding
+        mandatory input, removing input/output…
+* [ ] All inputs are used in process.
+* [ ] All outputs fields have their ``re-save``.
+
+## Additional guidelines
+
+### Processes ###
+
+* Code that was once accepted should be kept like it is. It should be changed
+    only when it's functionality is changed, or small parts may be changed
+    to fix bugs (If you change it only to make it nicer there is a chance
+    to introduce a bug).
+* Set the optimal number of cores that the tool can use (check if parallelization
+    can be performed).
+* Description of tool should be self explanatory with links to tool homepage
+    and publication.
+* Write explanatory input labels and add short commands in [ ] (for example:
```

Comment:
Write explanatory input labels and add command line parameter names in square brackets (for example: Maximum length [`--maxlength`]).